cg/meta/create_validation_cases/validation_case_data.py

```diff
@@ -0,0 +1,124 @@
+"""Model for down sampling meta data."""
+
+import logging
+from pathlib import Path
+
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import DataDelivery, Priority
+from cg.meta.create_validation_cases.validation_data_input import ValidationDataInput
+from cg.store.models import ApplicationVersion, Case, Sample
+from cg.store.store import Store
+from cg.utils.calculations import multiply_by_million
+
+LOG = logging.getLogger(__name__)
+
+
+class ValidationCaseData:
+    def __init__(
+        self, status_db: Store, hk_api: HousekeeperAPI, validation_data_input: ValidationDataInput
+    ):
+        """Initialize the validation sample data and perform integrity checks."""
+        self.status_db: Store = status_db
+        self.housekeeper_api: HousekeeperAPI = hk_api
+        self.input_data: ValidationDataInput = validation_data_input
+        self.case_id: str = validation_data_input.case_id
+        self.case_name: str = validation_data_input.case_name
+        self.original_case: Case = self.get_case_to_copy()
+        self.original_samples: list[Sample] = self.get_samples_to_copy()
+        self.validation_samples: list[Sample] = self.get_validation_samples()
+        self.validation_case: Case = self._generate_validation_case()
+        LOG.info(f"Validation Data checks completed for {self.case_id}")
+
+    @staticmethod
+    def _validation_sample_name(sample: Sample) -> str:
+        """Return a new validation sample identifier. The identifier removes "ACC" and prepends "VAL"."""
+        return sample.internal_id.replace("ACC", "VAL")
```

Comment:
We do have a stage user in lims so all orders submitted via the stage order portal will have "STG" instead of "ACC". Just adding an example.

Reply:
It will now take the first three letter and changes them to "VAL"... In cases where a sample has something else then a conventional id (3 letters and some numbers) this would be a bit iffy...

perhaps a "validation" suffix would be better?